wallabag/wallabag
# Bigger image preview in case of only image content

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | fix #1770
| License       | MIT

Display a bigger image in case of image content
If the content is only an image, we can display a bigger preview image because we won’t have text to display.

Preview (first card is image content only, then regular content, then content without preview):

![image](https://cloud.githubusercontent.com/assets/62333/20117635/bd47d2a2-a601-11e6-9002-2916f5548adf.png)

Also:
- split different card view to avoid too much complexity in the `entries.html.twig`
- Remove the left margin of tag (you can see on the third card of the screenshot)
- Add `grunt watch` task

Improvement (I failed to do it...) :
- make the image clickable
- then, remove the useless title
- but keep the domain name
